tools/pipelines/build-eslint-config-fluid.yml

```diff
@@ -92,6 +94,4 @@ extends:
     testCoverage: ${{ variables.testCoverage }}
     taskBuild: build
     taskBuildDocs: false
-    taskLint: false
-    taskTest:
-      - test
+    taskLint: false
```

Comment:
@alexvy86 @Josmithr 

Wanted to double check if removing `taskTest` is correct as the test-suites in `eslint-config-fluid` have been moved to the `eslint-plugin-fluid` package.

Reply:
It is correct, though I would recommend an alternative solution. I would recommend keeping this, and replacing the `test` script in the package with `echo "This package doesn't have any tests"`. That way if we add new tests later, the CI pipeline will automatically run them.

The reason I suggest this is that when we first added tests to the package a few months ago, none of us realized we needed to modify the pipeline to ensure they were running. So the tests weren't actually being run until I happened to notice that one of them was failing locally and did some digging.

Reply:
I'll just add it's actually not entirely correct 😄 , it needs to be specified as `taskTest: []` otherwise [the default](https://github.com/microsoft/FluidFramework/blob/bf7ae34b81cb28099a765311423071bbe697a830/tools/pipelines/templates/1ES/build-npm-package.yml#L30-L33) will take over. Example where we disable them [here](https://github.com/microsoft/FluidFramework/blob/bf7ae34b81cb28099a765311423071bbe697a830/tools/pipelines/build-build-common.yml#L96u).